Library/Homebrew/cask/spec/cask/artifact/uninstall_spec.rb

```diff
@@ -7,12 +7,24 @@
     Hbc::Artifact::Uninstall.new(cask, command: Hbc::FakeSystemCommand)
   }
 
+  let(:test_home_path) { TEST_FIXTURE_DIR }
+  let(:original_home_path) { ENV['HOME'] }
+  let(:absolute_path) { TEST_FIXTURE_DIR/'cask'/'absolute_path' }
+  let(:path_with_tilde) { TEST_FIXTURE_DIR/'cask'/'path_with_tilde' }
+  let(:glob_path1) { TEST_FIXTURE_DIR/'cask'/'glob_path1' }
+  let(:glob_path2) { TEST_FIXTURE_DIR/'cask'/'glob_path2' }
```

Comment:
I made this change because this test https://github.com/Homebrew/brew/blob/master/Library/Homebrew/cask/spec/cask/artifact/zap_spec.rb#L261 is also using fixtures and I thought that this is common approach to make such things in this project.